File: demo7/src/main/java/com/example/demo7/MainActivity.java
package com.example.demo7;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void test(View view){
        Toast.makeText(this,"test",Toast.LENGTH_SHORT).show();
//        startActivity(new Intent(this,Main2Activity.class));
        startActivity(new Intent().setAction("com.example.test"));
    }
}

File: task2/src/main/java/com/example/task2/MainActivity.kt
package com.example.task2

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val task: TextView =findViewById(R.id.taskid2)
        task.text= "task_id==$taskId"
    }

      public fun test(view: View){
          val intent:Intent=Intent(this,Main2Activity::class.java)
          startActivity(intent)
      }
}
